rules/linux/privilege_escalation_suspicious_uid_guid_elevation.toml

```diff
@@ -64,12 +64,17 @@ sequence by host.id, process.entity_id with maxspan=1s
      process.parent.executable : ("/tmp/newroot/*", "/opt/carbonblack*") or
      process.parent.executable in (
        "/opt/SolarWinds/Agent/bin/Plugins/JobEngine/SolarWinds.Agent.JobEngine.Plugin", "/usr/bin/vmware-toolbox-cmd",
-       "/usr/bin/dbus-daemon", "/usr/bin/update-notifier", "/usr/share/language-tools/language-options"
+       "/usr/bin/dbus-daemon", "/usr/bin/update-notifier", "/usr/share/language-tools/language-options",
+       "/opt/SolarWinds/Agent/*", "/usr/local/sbin/lynis.sh"
      ) or
-     process.executable : ("/opt/dynatrace/*", "/tmp/newroot/*") or
+     process.executable : ("/opt/dynatrace/*", "/tmp/newroot/*", "/opt/SolarWinds/Agent/*") or
```

Comment:
Side note: We should always consider supply chain compromises as well. However it's a double edged sword between FP volume and fidelity. Better suited for hunting.